### PR TITLE
Officer Cheapsky

### PR DESCRIPTION
**Pull request by Shadowmech88**
cheapsky_1
Adds Officer Cheapsky.

Officer Cheapsky can be constructed with a cardborg helmet, a remote signaler, a sharp item, and a proximity sensor.
Officer Cheapsky functions similarly to Officer Beepsky, but instead of stunning and cuffing criminals, he just scolds them.

:cl:
 * rscadd: Added a new secbot, Officer Cheapsky. Can be constructed with a cardborg helmet, a remote signaler, a sharp item, and a proximity sensor.

### CODE REVIEW

**Comment by Aurx** [comment]
Is this a cheap Italian restaurant?  
Because I just got served a plate full of copypasta.

**Comment by Kitfox88** [comment]
never stop, shadow

File: code/game/machinery/bots/secbot.dm [hunk 1]
```diff
@@ -876,3 +876,192 @@ Auto Patrol: []"},
...
+
+/obj/item/weapon/secbot_assembly/cheapsky/attackby(obj/item/weapon/W, mob/user)
+	if(W.sharpness && W.sharpness_flags & SHARP_BLADE && (!src.build_step))
+		src.build_step++
+		src.overlays += image('icons/obj/aibots.dmi', "bs_hole")
+		to_chat(user, "You cut a hole in \the [src]!")
+
+	else if(isprox(W) && (src.build_step == 1))
+		if(user.drop_item(W))
+			to_chat(user, "You complete the Securitron! Beep boop.")
```
Comment (kol1th) [inline code/game/machinery/bots/secbot.dm:1054]: >securitron

Comment (Shadowmech88) [reply code/game/machinery/bots/secbot.dm:1054]: Officer Cheapsky is *a* securitron. Though I suppose I could change it to `"You complete \the [S]!"`

Comment (Blithering) [reply code/game/machinery/bots/secbot.dm:1054]: A bot can dream can't he?

**Comment by Blithering** [comment]
Does it have any emagged function?

**Comment by Shadowmech88** [comment]
Currently unchanged from Beepsky, so I'd imagine it'd make him just start chasing random people.

**Comment by Kitfox88** [comment]
If he's emagged he starts cheering on lawbreakers instead.

**Comment by SonixApache** [comment]
[UPDATE WIKI]

**Comment by Zth--** [comment]
Pleasssse udpate wiki

**Comment by Shadowmech88** [comment]
Wiki updated.

**Pull request merged**